tests/test_epics_ipm.py

```diff
@@ -49,11 +69,14 @@ def test_ipm_motion(ipm):
 
 
 @using_fake_epics_pv
-def test_ipm_subscriptions(ipm):
+def test_ipm_subscriptions():
+    logger.debug('test_ipm_subscriptions')
+    ipm = fake_ipm()
     #Subscribe a pseudo callback
     cb = Mock()
     ipm.subscribe(cb, event_type=ipm.SUB_STATE, run=False)
     #Change the target state
     ipm.target.state._read_pv.put('OUT')
+    attr_wait_true(cb, 'called')
```

Comment:
`assert cb.called` was running before the subscription could call it?

Reply:
The new FakeEpicsPV is not guaranteed to return a callback instantly, much like a real epics PV.